server/index.js

```diff
@@ -1,9 +1,40 @@
-const express = require('express')
-const app = express()
- 
-app.get('/registration', function (req, res) {
-  console.log("sending Hello")
-  res.send('Hello')
+const express = require('express');
+const cors = require('cors');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser')
+
+// Create global app object
+const app = express();
+
+app.use(cors());
+
+// import admin and Registered routes
+const adminRoutes = require('./routes/admin')
+const RegisteredRoutes = require('./routes/registered');
+// const path = require('path/posix');
+
+// call body-parser and register a middleware by yielding with urlencoded
+// pass an option to be able to parse non-default feature 
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(bodyParser.json())
+
+// serve static files
+// app.use(express.static(path.join(__dirname, 'web')));
+
+// call other routes and filter paths
+app.use('/admin', adminRoutes);
+app.use(RegisteredRoutes);
+
+// handling 404 Error Page
+app.use((req, res, next) => {
+  res.status(404).send('Page not Found')
```

Comment:
The same reason with https://github.com/nishi-ai/softball-register/pull/6#discussion_r679267436. We don't need to take care of `Page nor found` error on the server. It can be done on `react` app side.